lib/skate/detours/detours.ex

```diff
@@ -376,6 +390,17 @@ defmodule Skate.Detours.Detours do
 
   defp update_swiftly(_, _, _), do: nil
 
+  defp get_swiftly_adjustment_from_response(adjustments_response, detour) do
+    adjustments_response
+    |> Map.get(:adjustments, [])
+    |> Enum.filter(fn adjustment ->
+      Map.get(adjustment, :status) == "ACTIVE" &&
+        Map.get(adjustment, :adjustmentType) == "DETOUR_V0" &&
+        Map.get(adjustment, :notes) == Integer.to_string(detour.id)
+    end)
+    |> Enum.at(0)
```

Comment:
nit/suggestion(non-blocking): just curious, is there a reason to prefer `Map.get` over pattern matching?

```suggestion
    detour_id = Integer.to_string(detour.id)

    adjustments_response
    |> Map.get(:adjustments, [])
    |> Enum.filter(fn
      %{status: "ACTIVE", adjustmentType: "DETOUR_V0", notes: ^detour_id} -> true
      _ -> false
    end)
    |> Enum.at(0)
```